### DISCUSSION TITLE
telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running

### DISCUSSION THREAD

**Discussion by tanakaakiosab** [MrMissx/Telegram_Forwarder#93] [General]
When I run the program, I see below alert.
I have already stop it and restart but the result is the same.
Can you advise how to fix ?

telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running

**Comment by MrMissx** [comment]
you could revoke and create a new token from botfather

**Reply by tanakaakiosab** [reply]
Thank you.
I will try.

**Comment by tanakaakiosab** [comment]
Can I ask?

If I want to forward channel A to B   
and Channel C to D.

Is this correct?


[
    {
        "source": A,
        "destination": [A, B]
    },
    {
        "source": C,
        "destination": [C, D]
    }
]

**Reply by MrMissx** [reply]
with that config it will be:
- Channel A will be forwarded to Channel A and B
- Channel C will be forwarded to channel C and D

if you don't want to forward to the same channel don't put it on the `destination` value

Here are example if you wanted to forward channel A to B and channel C to D
```
[
  {
    "source": A,
    "destination": [B]
  },
  {
    "source": C,
    "destination": [D]
  }
]
```
